app/templates/components/search-facet-registration-type.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="registration-type-selector">
+    <i class='type-selector-warning' style='color: red; display: none'>Only available with OSF Registries</i>
```

Comment:
This was intentional, but Product may have a different opinion now.  The demo was prior to the tooltip being merged, but the tooltip PR was in CR at the time, so we knew about it. We discussed moving the tooltip to the search widget in ember-osf, but they said they preferred just moving the red text under the heading.

More detail:
Each of these search facets are now their own component.  The search widget loops through all the search facets and prints their name (OSF Registration Type), and then renders the component beneath that name.  

So in the demo, Product agreed that just moving the red warning text inside the component instead of above it was sufficient. Otherwise, there needs to be some optional tooltip that could appear beside every search facet name.